core/controllers/learner_group.py

```diff
@@ -424,13 +420,10 @@ def get(self, learner_group_id: str) -> None:
             )
         )[learner_user_id]
 
-        users_settings = user_services.get_user_settings(learner_user_id)
         learner_progress = {
             'username': self.username,
             'progress_sharing_is_turned_on':
                 progress_sharing_permission,
-            'profile_picture_data_url':
```

Comment:
How are we going to make sure that removing this field does not breaks the whole functionality? Cause this would be resulting in an error on the frontend for sure.

Reply:
Yes you are right, please note that these changes are part of a complete project and for the detailed description and sequence of execution you can take a look at this [comment](https://github.com/oppia/oppia/pull/16584/files#r1034598191)